### ISSUE TITLE
2.7.0 - unable to create playlists - spams kodi.log - freezes kodi

### ISSUE THREAD

**Issue by joswart** [croneter/PlexKodiConnect#724]
With plexkodiconnectm2.7.0 I get an endless amount of notices and errors in my kodi.log:

```
09:03:46.964 T:3892306800  NOTICE: PLEX.playlists: Skipping creation of playlist 808035
09:03:47.021 T:3892306800   ERROR: PLEX.playlists.kodi_pl: Detected spanning tree issue, abort sync for {'plex_id': 812742, 'plex_name': 'Film_365', 'kodi_type': 'video', 'kodi_filename': 'None', 'kodi_path': 'None', 'plex_updatedat': 1550045026, 'kodi_hash': 'None'}
09:03:47.021 T:3892306800  NOTICE: PLEX.playlists: Skipping creation of playlist 812742
09:03:47.090 T:3892306800   ERROR: PLEX.playlists.kodi_pl: Detected spanning tree issue, abort sync for {'plex_id': 828127, 'plex_name': 'Film_365', 'kodi_type': 'video', 'kodi_filename': 'None', 'kodi_path': 'None', 'plex_updatedat': 1550045027, 'kodi_hash': 'None'}
09:03:47.091 T:3892306800  NOTICE: PLEX.playlists: Skipping creation of playlist 828127
09:03:47.159 T:3892306800   ERROR: PLEX.playlists.kodi_pl: Detected spanning tree issue, abort sync for {'plex_id': 805686, 'plex_name': 'Film_366', 'kodi_type': 'video', 'kodi_filename': 'None', 'kodi_path': 'None', 'plex_updatedat': 1550045027, 'kodi_hash': 'None'}
```

Whatever plexkodiconnect is doing here, eats my CPU and causes kodi to freeze. Downgrading to 2.6.5 (and resyncing) completely removed the issue.

I have seen this issue on multiple instances. One is an Ubuntu on an i3 processor, the other is a wetek hub running librelec 9.

Both running Leia.

**Comment by croneter** [comment]
Please share your entire debug log, see https://github.com/croneter/PlexKodiConnect/wiki/How-to-Report-A-Bug

**Comment by croneter** [comment]
What are the names of your playlists on the Plex side?

**Comment by joswart** [comment]
I don’t have any playlists on the Plex site. That’s what baffles me as well. 

Also downgraded all pkc’s, due to the family account issues. I will try again tomorrow with 2.7. 

Jord

**Comment by croneter** [comment]
That should be impossible. Please also share the entire debug log.

**Comment by joswart** [comment]
Right.... I do have a ton of Playlists.

I never created a single one myself... They all have exactly 17h31m of content, so they all probably have the same origin. Since I never ever look at playlists, I missed this. Currently deleting them 100 by 100.... will take a lot of time. 

Anyway: this seems to be my personal issue. Thanks for doubting me!

**Comment by colbyzg** [comment]
I ran into the same issue today (thankfully I found this posted) - somehow I have 77k random playlists. As a temp fix I disabled playlist syncing in the PKC settings. 

I found this script to help with deletion:
https://gist.github.com/blacktwin/45c420cbba4e18aadc8cc5090a67b9d1

**Comment by croneter** [comment]
Please follow https://github.com/croneter/PlexKodiConnect/wiki/How-to-automatically-delete-playlists

**Comment by joswart** [comment]
Deleting the playlists made my PMS grab all available memory, which caused my system to come to an absolute halt. There might be a memory leak there somewhere. Since the scripts above use the Plex API, there was no way to delete all the playlists.

I used the following method (highly not recommended, might break everything, make a backup before you even think about trying this, etc. etc.):

Stop Plex and navigate to the database directory.

`sqlite3 com.plexapp.plugins.library.db "DROP index 'index_title_sort_naturalsort'"
sqlite3 com.plexapp.plugins.library.db "DELETE from metadata_items where title like 'Film_%'"
sqlite3 com.plexapp.plugins.library.db "PRAGMA integrity_check"`

Start Plex.

Not sure if I cleaned out everything, I have the feeling only the listing of the playlist is deleted, not its actual content. If someone has more info here, I would be delighted to know!

Thanks agains for the support (and the fantastic Plugin, naturally!).
